snuba/cli/consumer.py

```diff
@@ -54,6 +54,12 @@ def consumer(raw_events_topic, replacements_topic, commit_log_topic, consumer_gr
     dataset_name = dataset
     dataset = get_dataset(dataset_name)
 
+    if not bootstrap_server:
+        if dataset_name in settings.DEFAULT_DATASET_BROKERS:
+            bootstrap_server = settings.DEFAULT_DATASET_BROKERS[dataset_name]
+        else:
+            bootstrap_server = settings.DEFAULT_BROKERS
```

Comment:
I guess this could have also as easily been `bootstrap_server = settings.DEFAULT_DATASET_BROKERS.get(dataset_name, settings.DEFAULT_BROKERS)` but apparently my brain wasn't firing on all cylinders yesterday afternoon.